src/main/java/dog/pawbook/logic/parser/AddDogCommandParser.java

```diff
@@ -52,8 +52,8 @@ protected String getUsageText() {
      * @throws ParseException if the user input does not conform the expected format
      */
     public AddDogCommand parse(String args) throws ParseException {
+        assert(!args.isEmpty());
```

Comment:
This is not something that is meant for an assertion, since it is one of the expected possible errors that we need to handle. Assertions are for verifying assumption that should not be broken and if they are indeed broken, the program is unable to handle and we need to investigate further. I suggest that this be removed.

Reply:
this needs to be removed